chapters/classes.tex

```diff
@@ -1478,7 +1478,7 @@ \subsection{Real Type}\label{real-type}
   parameter StringType quantity    = "";
   parameter StringType unit        = "" "Unit used in equations";
   parameter StringType displayUnit = "" "Default display unit";
-  parameter RealType min = -Inf, max = +Inf; // Inf denotes a large value
+  parameter RealType min = $\ldots$, max = $\ldots$; //  See below
```

Comment:
Perhaps the _See below_ goes without saying, and it would be of more interest to explain that these limit the value?
```suggestion
  parameter RealType min = $\ldots$; // Lower bound for $\langle$$\mbox{\emph{value}}$$\rangle$
  parameter RealType max = $\ldots$; // Upper bound for $\langle$$\mbox{\emph{value}}$$\rangle$
```

Also, if we now introduce the use of ellipsis for a default being explained in the text, shouldn't `fixed` be handled in the same style?
```
  parameter BooleanType fixed = $\ldots$; // Enforce exact value of 'start'
```
together with
> The default \lstinline!fixed!-attribute is \lstinline!true! for parameters and constants, and \lstinline!false! for other variables.

Reply:
Sure